documents/stipendium_initiativ.tex

```diff
@@ -35,9 +35,9 @@
   \url{www.pep-dortmund.org}
 }
 \ofoot{%
-	Henning Moldenhauer CP-01-190\\
-	Alex Birnkraut CP-03-108\\
-	Vanessa Müller CP-03-107%
+	Christian Arauner P2-01-423\\
+	Rene-Marcel Lehner P2-01-423\\
```

Comment:
Same office for convenience?

Reply:
as the Rene has no office, I just chose yours